akka-docs/src/main/paradox/stream/operators/StreamConverters/asJavaStream.md

```diff
@@ -20,6 +20,17 @@ The Java 8 `Stream` will be ended when the stream flowing into this `Sink` compl
 
```

Comment:
Above:
Change "Java `Stream` throws exception in case reactive stream failed."
To "If the Java `Stream` throws an exception, the Akka Stream is cancelled."